classes/mail/variables/RecipientEmailVariable.inc.php

```diff
@@ -65,30 +65,22 @@ protected function values() : array
      * Array containing full names of recipients in all supported locales separated by a comma
      * @return array [localeKey => fullName]
      */
-    protected function getRecipientsFullName() : array
+    protected function getRecipientsFullName(string $locale): string
     {
-        $fullNamesLocalized = [];
-        $supportedLocales = PKPLocale::getSupportedLocales();
```

Comment:
`PKP\i18n\PKPLocale` import may be removed here and in some other Variable classes. This is what I often forget :)

Reply:
Interesting because the code formatter is supposed to automatically remove unused import statements. I've seen it work too... :thinking: